modules/jolt_physics/objects/jolt_soft_body_3d.cpp

```diff
@@ -145,7 +145,7 @@ bool JoltSoftBody3D::_ref_shared_data() {
 		LocalVector<int> &mesh_to_physics = iter_shared_data->value.mesh_to_physics;
 
 		JPH::SoftBodySharedSettings &settings = *iter_shared_data->value.settings;
-		settings.mVertexRadius = JoltProjectSettings::soft_body_point_radius;
+		settings.mVertexRadius = point_radius * JoltProjectSettings::soft_body_point_radius_factor;
```

Comment:
The `SoftBodySharedSettings` object is an object that is shared amonst all soft bodies using the same mesh (see `mesh_to_shared` above). This means that with this implementation you can get into a situation where modifying the point radius of one soft body accidentally may modify another. I should probably move `mVertexRadius` to `SoftBodyCreationSettings` instead.

Reply:
That makes sense. It's part of the library so I can't do the change here directly.

Reply:
If you take https://github.com/jrouwe/JoltPhysics/commit/f3d906f8c0a07a6993ecb2ff962c892a04843daa (or upgrade all of Jolt to the latest version - there shouldn't be any breaking changes) then you should be able to fix it.